include/deal.II/base/tensor_accessors.h

```diff
@@ -561,8 +563,10 @@ namespace TensorAccessors
     struct ExtractHelper<rank, rank>
     {
       template<typename T, typename ArrayType>
-      inline static
-      T &extract(T &t, const ArrayType &indices)
+      inline
+      static
+      T &extract(T &t,
+                 const ArrayType &)
       {
         return t;
       }
```

Comment:
Can you do me a favor and also fix above comment of mine? It states "For dimension == rank" - it must read "For position == rank".